cpp/src/cugraph.cu

```diff
@@ -383,3 +386,77 @@ gdf_error gdf_bfs(gdf_graph *graph, gdf_column *distances, gdf_column *predecess
   bfs.traverse(start_node);
   return GDF_SUCCESS;
 }
+
+gdf_error gdf_jaccard(gdf_graph *graph, void *c_gamma, gdf_column *weights, gdf_column *weight_j) {
+  
+  GDF_REQUIRE(graph->adjList != nullptr || graph->edgeList != nullptr, GDF_INVALID_API_CALL);
```

Comment:
It seems to me that the code works only when `graph->adjList != nullptr`.

Reply:
if `graph->adjList is nullptr`, the next line will add the adjacency list from edge list